media/css/externalpages/pocket/includes/_nav.scss

```diff
@@ -185,12 +185,8 @@ $image-path: '/media/img/externalpages/pocket';
         display: none;
     }
 
-    &:active {
+    &:active::before {
```

Comment:
Why not just remove this whole selector instead?

Reply:
Because we want to keep the `active` pseudo element style change? I didn't author the existing nav styles, just didn't want to remove anything that's not strictly related to fixing the gub.

Reply:
Sounds good, I can discuss with Nathan 👍🏼

Reply:
I think we should still keep the selector, which is in line with @alexgibson's change. It should be the same as the style above the selector:
```
   &:hover::before {
        display: none;
    }
```